Fix/sla details on where penalties go

davidsiska-vega opened the pull request.

davidsiska-vega committed:
fix: clarify that spot doesn't have insurance pool and where penalties go

davidsiska-vega commented on protocol/0044-LIME-lp_mechanics.md:
```diff
@@ -292,9 +295,12 @@ Note:
 
 - If a liquidity provider has `fraction_of_time_on_book` >= `market.liquidity.committmentMinTimeFraction`, no penalty will be taken from their bond account (<a name="0044-LIME-013" href="#0044-LIME-013">0044-LIME-013</a>)
-- If a liquidity provider has `fraction_of_time_on_book` = `0.3`, `market.liquidity.committmentMinTimeFraction = 0.6`, `market.liquidity.sla.nonPerformanceBondPenaltySlope = 0.7`, `market.liquidity.sla.nonPerformanceBondPenaltyMax = 0.6` at the end of an epoch then they will forfeit `35%` of their bond stake, which will be transferred into the market's insurance pool (<a name="0044-LIME-014" href ...
+- If a liquidity provider has `fraction_of_time_on_book` = `0.3`, `market.liquidity.committmentMinTimeFraction = 0.6`, `market.liquidity.sla.nonPerformanceBondPenaltySlope = 0.7`, `market.liquidity.sla.nonPerformanceBondPenaltyMax = 0.6` at the end of an epoch then they will forfeit `35%` of their bond stake, which will be transferred into the market's insurance pool (<a name="0044-LIME-014" href ...
```
```suggestion
- If a liquidity provider has `fraction_of_time_on_book` = `0.3`, `market.liquidity.committmentMinTimeFraction = 0.6`, `market.liquidity.sla.nonPerformanceBondPenaltySlope = 0.7`, `market.liquidity.sla.nonPerformanceBondPenaltyMax = 0.6` at the end of an epoch then they will forfeit `35%` of their bond stake, which will be transferred into the market's insurance pool (<a name="0044-LIME-014" href="#0044-LIME-014">0044-LIME-014</a>)
```

witgaw commented on protocol/0044-LIME-lp_mechanics.md:
```diff
@@ -306,10 +312,15 @@ Note:
-- For a market with `market.liquidity.earlyExitPenalty = 0.25` and `total stake = target stake + 140` already, if the following transactions occur:
+In the case of spot markets it will be transferred into the network treasury for the asset (<a name="0044-LIME-032" href="#0044-LIME-032">0044-LIME-032</a>)
+
+- For a futures market with `market.liquidity.earlyExitPenalty = 0.25` and `total stake = target stake + 140` already, if the following transactions occur:
```
This may need it's own code or at least the removal of the empty line to make approbation CI check happy.

gordsport replied:
@witgaw - I have sorted the AC code numbers on this and approbation is now passing. 

Can you please re-review?

gordsport committed:
fix: duplicated AC code

gordsport committed:
fix: remove duplicated AC

dupe AC: In the case of spot markets it will be transferred into the network treasury for the asset

gordsport committed:
fix: more duplicate ACs

gordsport committed:
fix: AC code errors
